common/rest/signals/receivers.py

```diff
@@ -0,0 +1,96 @@
+from django.utils import timezone
+from django.db.models.signals import pre_save, post_save
+from django.dispatch import receiver
+from django.conf import settings
+from common.rest.middleware.changeLog import get_current_request
+from elasticsearch import Elasticsearch
+from common.celery_app import app
+
+
+CHANGE_LOG_CREATE = "create"
+CHANGE_LOG_UPDATE = "update"
+
+def skip_agent_model(sender):
+    return sender._meta.model_name == 'agent'
+
+def skip_agent_related_fields(field):
+    return field.related_model and field.related_model._meta.model_name == 'agent'
+
+@receiver(pre_save)
+def capture_old_values(sender, instance, **kwargs):
+    pass
+    # if skip_agent_model(sender):
+    #     return
+
+    # if instance.pk:
+    #     try:
```

Comment:
remove the code already commented

Reply:
I've removed the commented code as requested.